ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/MessageDrivenComponentDescriptionFactory.java

```diff
@@ -103,7 +103,7 @@ private void processMessageBeans(final DeploymentUnit deploymentUnit, final Coll
         for (final AnnotationInstance messageBeanAnnotation : messageBeanAnnotations) {
             final AnnotationTarget target = messageBeanAnnotation.target();
             final ClassInfo beanClassInfo = (ClassInfo) target;
-            if (! MdbValidationsUtil.assertMDBClassValidity(beanClassInfo).isEmpty() ) {
+            if (! EjbValidationsUtil.assertEjbClassValidity(beanClassInfo).isEmpty() ) {
```

Comment:
Since this seems to only be used for MDBs, why the rename?

Reply:
@kabir Because this PR aims at extending the validation did to Mdb to Ejb! :)